sgv2-restapi/src/main/java/io/stargate/sgv2/restsvc/resources/Sgv2RowsResourceImpl.java

```diff
@@ -72,45 +72,42 @@ public Response getRowWithWhere(
       throw new WebApplicationException(e.getMessage(), Status.BAD_REQUEST);
     }
 
-    return callWithTable(
-        bridge,
-        keyspaceName,
-        tableName,
-        false,
-        (tableDef) -> {
-          final ToProtoConverter toProtoConverter = findProtoConverter(tableDef);
-
-          final List<BuiltCondition> whereConditions;
-          try {
-            whereConditions = new WhereParser(tableDef, toProtoConverter).parseWhere(where);
-          } catch (IllegalArgumentException e) {
-            throw new WebApplicationException(e.getMessage(), Status.BAD_REQUEST);
-          }
-
-          final QueryOuterClass.Query query;
-          if (columns.isEmpty()) {
-            query =
-                new QueryBuilder()
+    QueryOuterClass.Response response =
+        queryWithTable(
+            bridge,
+            keyspaceName,
+            tableName,
+            tableDef -> {
+              final ToProtoConverter toProtoConverter = findProtoConverter(tableDef);
+
+              final List<BuiltCondition> whereConditions;
+              try {
+                whereConditions = new WhereParser(tableDef, toProtoConverter).parseWhere(where);
```

Comment:
This is probably where we would need to make `WhereParser` indicate things like missing table/column
using specific exception, to allow schema fetch. Although maybe it's not a big issue for this particular query:
the case of adding a column and then immediately referencing column might not be common?
Probably more common with row insert/create/update case.

Reply:
It sounds like retry on exception from query building already results in retry so that should be good. I'll go over PR again to see it, assuming it's all fine.